### PR TITLE
[v2] tests: fix pause tests

### PR DESCRIPTION
**Pull request by giuseppe**
adapt the tests to work with Podman v2.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: test/utils/utils.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func (p *PodmanTest) NumberOfPods() int {
 // This function assumes only one container is active.
 func (p *PodmanTest) GetContainerStatus() string {
 	var podmanArgs = []string{"ps"}
-	podmanArgs = append(podmanArgs, "--all", "--format={{.Status}}")
+	podmanArgs = append(podmanArgs, "--all", "--format={{.State}}")
```
Comment (mheon) [inline test/utils/utils.go:170]: This actually seems pretty significant - is this a breaking change from Docker?

Comment (giuseppe) [reply test/utils/utils.go:170]: yes, it is a breaking change with Docker

Comment (mheon) [reply test/utils/utils.go:170]: I think we need to either alias `Status` to `State` or change it back completely, then

Comment (jwhonce) [reply test/utils/utils.go:170]: We could rename:
https://github.com/containers/libpod/blob/master/cmd/podman/containers/ps.go#L268 to Status() or add a Status() method that calls State().  /cc @baude Was State() chosen for a reason?

Comment (giuseppe) [reply test/utils/utils.go]: thanks for the hint, I will add a function Status() that calls State() for now.

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by giuseppe** [comment]
@TomSweeneyRedHat can you use "/" + "lgtm" ?

**Pull request merged**